float32 goal_distance
float32 goal_angle
---
bool success
